src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/ItemsJson.java

```diff
@@ -11,4 +11,7 @@ public class ItemsJson {
 
     @Expose
     public Map<String, Integer> crimson_tiers;
+
+    @Expose
+    public List<String> trophy_armors;
```

Comment:
You can use the `NEUInternalName` class here, to indicate these Strings are item ids.

```suggestion
    public List<NEUInternalName> trophy_armors;
```